src/main.py

```
@@ -0,0 +1,142 @@
+async def transcribe_from_s3(
+    path: str,
+    flow: 'Flow' = Depends(aqueduct_flow),
+    ap: AudioProcessor = Depends(audio_processor),
+    s3: S3Processor = Depends(s3_processor),
+):
+    audio_files = s3.list_input_audio_files(path)
+    if not audio_files:
+        raise HTTPException(status_code=404, detail='No audio files found')
+
+    transcribe_id = str(uuid.uuid4())
+    asyncio.create_task(transcribe_audio_files(flow, ap, s3, audio_files, transcribe_id))
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)